src/schema/query/resource.query.ts

```diff
@@ -22,6 +23,18 @@ export default {
     const ability: AppAbility = user.ability;
     const resource = await Resource.findOne({ _id: args.id });
 
+    let index = 0;
+    for await (const field of resource.fields) {
```

Comment:
@ketanreliefapps we should not put that there, but in the resolver of the resource.type + form.type

Reply:
I think the logic is good, but it cannot work for example if you query multiple resources at same time

Reply:
i have added in the meta resolver